engine/inc/com/centreon/engine/contact.hh

```diff
@@ -61,7 +61,8 @@ class contact {
   // Base properties.
   std::string const& get_address(int index) const;
   std::vector<std::string> const& get_addresses() const;
-  void set_addresses(std::vector<std::string> const& addresses);
+  void set_addresses(const std::vector<std::string>& addresses);
+  void set_addresses(std::vector<std::string>&& addresses);
```

Comment:
in case on new conf, remove it and replace const std::vector<std::string>& in add_contact by std::vector<std::string>&&